scss/components/_steps.scss

```diff
@@ -0,0 +1,83 @@
+@charset "UTF-8";
+
+/*——————————————————*\
+  #STEPS WRAPPER
+\*——————————————————*/
+
+.steps {
+  width: 100%;
+  display: inline-block;
+  position: relative;
+}
+
+
+
+
+
+/*----------------------------------------------------------------------------*\
+  #STEP
+
+  .step (--current)
+
+\*----------------------------------------------------------------------------*/
+
+/**
+* 1 - Define margin between steps;
+* 2.1 - Vertical margin is half of a step -1px that's the line height;
+* 2.2 - The horizontal margin is half of the difference between the step wrapper and
+        the step content;
+* 3 - Get all next siblings from `.step--current`;
+*/
+
+$stepWidth: $grid*6;
+$beforeWidth: $grid*3;
```

Comment:
since it has same width and height, should be more semantic if you name this variable as `(step|before)Size`